src/AudioUnit/AUGraph.cs

```diff
@@ -162,7 +162,7 @@ public AudioUnitStatus RemoveRenderNotify (RenderDelegate callback)
 
 		HashSet<RenderDelegate> graphUserCallbacks = new HashSet<RenderDelegate> ();
 
-		[MonoPInvokeCallback(typeof(CallbackShared))]
+		[MonoPInvokeCallback (typeof(CallbackShared))]
```

Comment:
The rule I'm going with is "if the entire file is just space changes I skip it, else I accept them since I'm already screwing with file history".

I can be more careful with whitespace diffs if desired.